Booktrix/Extensions/UIViewController+Storyboard.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  UIViewController+Storyboard.swift
+//  Booktrix
+//
+//  Created by Impresyjna on 14.05.2017.
+//  Copyright © 2017 Impresyjna. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController {
+    func pushViewFromStoryboard(storyboardName: String, viewName: String) {
+        self.navigationController?.pushViewController(UIStoryboard(name: storyboardName, bundle: nil).instantiateViewController(withIdentifier: viewName) as UIViewController, animated: true)
+        
+    }
+    
+    func presentViewFromStoryboard(storyboardName: String, viewName: String) {
+        let storyBoard: UIStoryboard = UIStoryboard(name: storyboardName, bundle: nil)
+        let newView = storyBoard.instantiateViewController(withIdentifier: viewName)
+        self.present(newView, animated: true, completion: nil)
+    }
+    
+    func hideKeyboardWhenTappedAround() {
+        let tap: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(UIViewController.dismissKeyboard))
```

Comment:
to `UIViewController.` i deklaracja typu `UITapGestureRecognizer` przy `let` chyba nie są potrzebne?